mobile/lib/src/presentation/pages/sleep_history_page/sleep_history_page.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:polydodo/src/application/sleep_history/sleep_history_cubit.dart';
+import 'package:polydodo/src/application/sleep_history/sleep_history_state.dart';
+import 'package:polydodo/src/presentation/navigation/navdrawer_widget.dart';
+import 'package:polydodo/src/presentation/navigation/routes/router.gr.dart';
+
+class SleepHistoryPage extends StatelessWidget {
```

Comment:
J'ai un problème avec le UX de la navigation de cette page. Quand tu choisis "History" y a pas moyen de revenir au menu précédent (y a pas le drawer en gros, which is bad). Quand tu chosis une entrée du history, t'as le bouton du drawer qui apparait (which is good).

Reply:
J'ai pas encore travaillé tant que ça la page d'historique, beaucoup de changements à faire. Tho le bouton est là, mais j'ai oublié de changer sa couleur donc c'est blanc sur blanc 😛